### PR TITLE
Raspberrypi service updates

### PR DESCRIPTION
**Pull request by YoDaMa**
these improvements add more documentation for setting up a daemon. Additionally they adjust the raspberrypi yaml to reboot after 10 minutes so the nightly builds can work well.

### CODE REVIEW

File: jenkins/raspberrypi/devops.service [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+After=multi-user.target
+
+[Service]
+# Client Env Vars
+# /etc/devops-environment is a file containing the necessary credentials for running E2E Tests 
+EnvironmentFile=/etc/devops-environment
+
+User=pi
+Type=idle
+ExecStart=/bin/bash /home/pi/myagent/run.sh > /home/pi/myagent/run.log 2>&1
```
Comment (massand) [inline jenkins/raspberrypi/devops.service]: I think your script doesn't install the agent, so you might want to add a comment with a link to the DevOps agent install steps to get this path - /home/pi/myagent

File: jenkins/raspberrypi/raspi.yaml [hunk 2]
```diff
@@ -106,11 +106,16 @@ jobs:
       artifactName: drop
       itemPattern: 'drop/**/*e2e_exe'
 
+  - bash: |
+      (sleep 1200; sudo reboot ) &
```
Comment (massand) [inline jenkins/raspberrypi/raspi.yaml]: 1200? I was thinking more like 60 seconds.

File: jenkins/raspberrypi/devops.service (hunk 1 above)
Comment (YoDaMa) [reply jenkins/raspberrypi/devops.service]: my script installs the devops agent but I've added more explicit comments in the setup_pi_for_e2e_tests.sh

File: jenkins/raspberrypi/raspi.yaml (hunk 2 above)
Comment (YoDaMa) [reply jenkins/raspberrypi/raspi.yaml]: reduced time

File: jenkins/raspberrypi/devops_service.sh [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+# single quotes are used to turn off special meaning of $ in variables.
+export IOTHUB_CONNECTION_STRING=''
+export IOTHUB_EVENTHUB_CONNECTION_STRING=''
+export IOTHUB_EVENTHUB_LISTEN_NAME=''
+export IOTHUB_E2E_X509_THUMBPRINT=''
+export IOTHUB_E2E_X509_CERT_BASE64=''
+export IOTHUB_E2E_X509_PRIVATE_KEY_BASE64=''
+export IOTHUB_EVENTHUB_CONSUMER_GROUP='$Default'
+IOTHUB_PARTITION_COUNT=16
```
Comment (jebrando) [inline jenkins/raspberrypi/devops_service.sh]: Add Export

**Pull request merged**